crates/prof/src/lib.rs

```diff
@@ -135,6 +136,18 @@ impl MetricDb {
 
             // Fill table rows
             for (label_values, metrics) in metrics_dict {
+                // cycle_tracker_span is a long string, we don't need to display it in markdown
+                let label_values = label_values
+                    .iter()
+                    .enumerate()
+                    .map(|(i, val)| {
+                        if label_keys[i] == "cycle_tracker_span" {
+                            ""
+                        } else {
+                            val.as_str()
+                        }
+                    })
+                    .collect_vec();
```

Comment:
won't this still have a row for each cycle tracker span?

Reply:
yea - do we just not want those tables at all?

Reply:
removed the table entirely